### DISCUSSION TITLE
Basic  Setup (Compose File)

### DISCUSSION THREAD

**Discussion by fschofield** [advplyr/audiobookshelf#2709] [General]
Like to set up basic ABS on windows 11 on the C drive.   How do I configure the compose file?

This is what I tried to run 

version: "3.7"
services:
  audiobookshelf:
    image: ghcr.io/advplyr/audiobookshelf:latest
    ports:
      - 13378:80
    volumes:
      - C:/docker/abs/audiobooks
      - C:/docker/abs/config
      - C:/docker/abs/metadata
      - C:/docker/abs/ebooks
      
 error location must be absolute file path.   

Any help be great.   thanks

**Comment by DieselTech** [comment]
You removed too much from the volumes section. You have to keep the `:` and everything to the right of it.

**Reply by fschofield** [reply]
I thought I had it right by following the following example

volumes:
      - </path/to/audiobooks>:/audiobooks
      - </path/to/podcasts>:/podcasts
      - </path/to/config>:/config
      - </path/to/metadata>:/metadata

I replaced </path/to/auidobooks>:/auidobooks   with

- C:/docker/abs/auidobooks

**Reply by fschofield** [reply]
Ok I got her working looked at the example in the install guide again and so far so good thanks for your help.
